build.mill

```diff
@@ -591,6 +591,10 @@ trait MillStableScalaModule extends MillPublishScalaModule with Mima {
       "mill.main.MainModule.mill$define$BaseModule0$_setter_$evalWatchedValues_="
     ),
 
+    // Overrides added for new methods, ought to be safe
+    ProblemFilter.exclude[ReversedMissingMethodProblem]("mill.scalalib.JavaModule.mill$scalalib$JavaModule$$super$internalRepositories"),
+    ProblemFilter.exclude[ReversedMissingMethodProblem]("mill.scalanativelib.ScalaNativeModule.mill$scalanativelib$ScalaNativeModule$$super$coursierProject"),
+
```

Comment:
@alexarchambault these turn out not to be safe, we probably need to fix the code to not add calls to `super` since they are not binary compatible


CC @sake92

Reply:
I'm not sure I said it before, but I suggest to generate the access to `super.<overridden>` from our macros as soon as we can break bin-compat, so that we better prepare the binary code for ad-hoc overriding of tasks. It can be left unused, so it should not affect functionality, but we at least we trigger the super-accessors to be present in the binary code.

Reply:
Trying to address that in https://github.com/com-lihaoyi/mill/pull/4380, although the workarounds aren't fully satisfying IMO (they assume users don't add overrides between the trait defining the method and the one overriding it)

Reply:
Other such changes adding calls to `super` have been introduced in the past, apparently:
https://github.com/com-lihaoyi/mill/blob/9815c3548a0ec05d90be659480757f185e2f2960/build.mill#L523-L580